dali/operators/image/remap/remap.cu

```diff
@@ -47,15 +47,17 @@ class RemapGpu : public Remap<GPUBackend> {
     km_.Resize<Kernel>(1, spec_.template GetArgument<int>("device_id"));
     kernels::KernelContext ctx;
     ctx.gpu.stream = ws.stream();
+    dali::kernels::DynamicScratchpad scratchpad({}, ws.stream());
+    ctx.scratchpad = &scratchpad;
```

Comment:
Is that needed,

Reply:
No, but it will hopefully result in fewer upstream allocations. If I don't do that, KernalManager will create _another_ tempraroy DynamicScratchpad and use it for the kernel run.

Reply:
Understood.